backend/src/main/resources/db-migrations/common/v1-[common]-data-init.sql

```diff
@@ -0,0 +1,5 @@
+INSERT INTO SUBSCRIPTION 
+  (id, user_id, email, registered, subscribed, preferred_language, alert_type, created_by, created_date, last_modified_by, last_modified_date)
+VALUES
+  ('10001', '800011819', 'example@gmail.com', true, true, 1033, 'cdcp', 'flyway-community-edition', CURRENT_TIMESTAMP, 'flyway-community-edition', CURRENT_TIMESTAMP),
+  ('10002', '800011819', 'example@gmail.com', true, false, 1033, 'ei', 'flyway-community-edition', CURRENT_TIMESTAMP, 'flyway-community-edition', CURRENT_TIMESTAMP);
```

Comment:
We generally use UUIDs for IDs so that they are more difficult to guess.

Just use `uuidgen` to create a couple and paste those in there.

Reply:
ex:

```
❯ uuidgen
a6ea4925-f813-493e-80ec-a5b90ca28b6c
❯ uuidgen
d3419bcb-5e46-4678-831b-c9211d479429
```

Reply:
updated